awx/ui_next/package.json

```diff
@@ -74,6 +74,7 @@
     "lint": "eslint --ext .js --ext .jsx .",
     "add-locale": "lingui add-locale",
     "extract-strings": "lingui extract",
+    "extract-template": "lingui extract-template",
```

Comment:
Thanks to [this PR](https://github.com/lingui/js-lingui/pull/814), we can now create the `.pot` file.